server/README.md

```diff
@@ -41,6 +41,9 @@ $ docker run -v <your local configuration file path>:/scalardb/server/database.p
 # For custom log configuration
 $ docker run -v <your local configuration file path>:/scalardb/server/database.properties -v <your custom log4j2 configuration file path>:/scalardb/server/log4j2.properties -d -p 60051:60051 -p 8080:8080 ghcr.io/scalar-labs/scalardb-server:<version>
 
+# For set in environment variables
```

Comment:
```suggestion
# Use environment variables
```

Reply:
I have corrected the problem.
https://github.com/scalar-labs/scalardb/pull/376/commits/b66edbc35965d62fb205e1645d415764db63e9aa